Domain.Sql/DbContextExtensions.cs

```diff
@@ -132,6 +132,17 @@ public static void CreateAzureDatabase(this DbContext context, int dbSizeInGB, s
             context.WaitUntilDatabaseCreated();
         }
 
+        public static void CreateReadonlyUser(this DbContext context, DbReadonlyUser readonlyUser)
+        {

```

Comment:
Readonly --> ReadOnly.

Also, the readonlyUser parameter should have a null check.